### PR TITLE
bitcoin-explorer: drop project in favor of mempool

### PR DESCRIPTION
**Pull request by EmbeddedAndroid**
## Summary of Changes

Drop the `bitcoin-explorer` from the devenv now that https://github.com/stacks-network/sbtc/pull/233 has been merged

Implements: https://github.com/stacks-network/sbtc/issues/242

### Risks

Risk as low, as we have replaced this explorer

### How were these changes tested?

Locally, using devenv

### What future testing should occur?

Eventually, integration of the end to end testing would be ideal.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by friedger — APPROVED** [review]
LGTM

File: devenv/docker-compose.yml [hunk 1]
```diff
@@ -217,7 +189,7 @@ services:
       btcRpcUser: devnet
       btcRpcPwd: devnet
       network: testnet
-      bitcoinExplorerUrl: http://bitcoin-explorer:3002/api
+      bitcoinExplorerUrl: http://mempool-web:8083
```
Comment (AshtonStephens) [inline devenv/docker-compose.yml:192]: Got it - so the api formatting isn't relevant?

**Comment by AshtonStephens** [comment]
Trying locally.

**Comment by AshtonStephens** [comment]
Updated sbtc-docs documentation here: https://github.com/stacks-network/sbtc-docs/pull/98/files?short_path=9adbbfc#diff-9adbbfc18ce00420b65a70e59f1e19e278842fb5862d15d5ae68b386070f33d8

**Review by AshtonStephens — APPROVED** [review]
Works. Approved

**Pull request merged**